### DISCUSSION TITLE
High availability, production deployment. Documented example doesn't work

### DISCUSSION THREAD

**Discussion by jcleve** [DefectDojo/django-DefectDojo#10536] [Q&A]
I'm rolling out defectdojo and need an HA deployment with postresql and redis on k8s. The example in the [readme](https://github.com/DefectDojo/django-DefectDojo/blob/master/readme-docs/KUBERNETES.md) doesn't work. Celery, redis and postgres all come up with a single pod. Anyone have a working example/docs/notes/etc? I do have a minimal deployment up and working, but need HA to meet our prod requirements. A point in the right direction would be highly appreciated. Thanks!

**Comment by mykhailo-sindieiev** [comment]
Hello! How do you deploy defectdojo? If you deploy it with `helm`, could you share your `values.yaml` file?
